packages/xrpl/src/models/ledger/AMM.ts

```diff
@@ -0,0 +1,90 @@
+import { Currency } from '../common'
+
+import BaseLedgerEntry from './BaseLedgerEntry'
+
+interface AuthAccount {
+  AuthAccount: {
+    account: string
+  }
+}
+
+interface VoteSlot {
```

Comment:
Should these intermediate interfaces be exported?
cc @ckniffen

Reply:
Oh good catch. `AuthAccount` should be exported because it's used in `AMMBid` transaction. `VoteSlot` isn't used in a transaction.

Reply:
Probably both should.